manifests/pulp.pp

```diff
@@ -52,14 +65,28 @@
     repo_auth              => true,
     puppet_wsgi_processes  => 1,
     enable_katello         => true,
+    https_cert             => $https_cert,
+    https_key              => $https_key,
+    ca_cert                => $ca_cert,
     subscribe              => Class['certs', 'certs::qpid_client'],
   }
 
   contain ::pulp
 
-  foreman::config::passenger::fragment { 'pulp':
-    content     => file('katello/pulp-apache.conf'),
-    ssl_content => file('katello/pulp-apache-ssl.conf'),
+  if $manage_httpd {
```

Comment:
I just noticed, this is missing `/pub` for the http (80) vhost.

Reply:
Could you have a look at this again?